docs/api-reference/next.config.js/exportPathMap.md

```diff
@@ -4,7 +4,7 @@ description: Customize the pages that will be exported as HTML files when using
 
 # exportPathMap
 
-> This feature is exclusive of `next export`. Please refer to [Static HTML export](/docs/advanced-features/static-html-export.md) if you want to learn more about it.
+> This feature is meant to be used in conjunction with `next export`. Please refer to [Static HTML export](/docs/advanced-features/static-html-export.md) if you want to learn more about it.
```

Comment:
This change is not required, the feature is indeed exclusive of `next export`, it also works with `next dev` but it will only work when exporting the app, and the way you build an app when using `next export` is a bit different from `next build && next start`

```suggestion
> This feature is exclusive of `next export`. Please refer to [Static HTML export](/docs/advanced-features/static-html-export.md) if you want to learn more about it.
```